src/IntercomResources.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Intercom;
+
+abstract class IntercomResources
```

Comment:
I am wondering if this should be `IntercomResources` or `IntercomResource`. I tend to prefer singular, even though all the child classes are in plural 😅 What do you think @lucasmichot?

Reply:
Indeed. Changed in 635afc5f6c76f312289df9e0e9c4cf8d08fa1e2a